WD-Firmware/demo/build/SConscript_comrv_baremetal

```diff
@@ -46,6 +45,11 @@ listAsimCompilerDirectivs = [] + Env['A_FLAGS']
 
 # compilation defines (-D_)
 listCompilationDefines = [] + Env['PUBLIC_DEF']
+'''

```

Comment:
can we take all ComRV "configurations" to a file ? (D_COMRV_EVICTION_POLICY, D_COMRV_VERIFY_INIT_ARGS,etc ....)
to not be part of the build system.
this way you have file with default values and user can changed this file if needed. keep in mind this is a lib that will not be compiled all the time.

Reply:
done in 09b4fb8